### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by degville**
- provides a little more background information
- adds links to wider documentation
- a few small edits, updates and fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,68 @@
...
+and maintains installed snaps. 
+
+Snaps are app packages for desktop, cloud and IoT that update automatically,
+are easy to install, secure, cross-platform and dependency-free. They're being
+used on millions of Linux systems every day.
+
+Alongside its multifarious service and management functions, snapd:
+- implements the confinement policies that isolate snaps from the base system
+  and from each other
+- provides the _snap_ command that's used to install and remove snaps and
```
Comment (anonymouse64) [inline README.md]: I would put this item first, it's more generically applicable to visitors who don't know about snap confinement and isolation and things, they might just know snaps as that thing with the `snap` command

File: README.md [hunk 2]
```diff
@@ -1,38 +1,68 @@
...
 
-## Reporting bugs
+This is an [open source](COPYING) project and we warmly welcome community
+contributions, suggestions, and constructive feedback. If you're interested in
+contributing, please take a look at our [Code of Conduct](CODE_OF_CONDUCT.md)
+first.
 
-If you have found an issue with the application, please [file a bug](https://bugs.launchpad.net/snappy/+filebug) on the [bugs list on Launchpad](https://bugs.launchpad.net/snappy/).
+- to report an issue, please file [a bug
+  report](https://bugs.launchpad.net/snappy/+filebug) on our [Launchpad issue
```
Comment (anonymouse64) [inline README.md]: do we want the snapd project instead of snappy here?

File: README.md [hunk 3]
```diff
@@ -1,38 +1,68 @@
...
+# Welcome to snapd
+
+This is the code repository for **snapd**, the background service that manages
+and maintains installed snaps. 
+
+Snaps are app packages for desktop, cloud and IoT that update automatically,
+are easy to install, secure, cross-platform and dependency-free. They're being
+used on millions of Linux systems every day.
+
+Alongside its multifarious service and management functions, snapd:
```
[the remark this answers is not in the document]
Comment (degville) [reply README.md]: You're right - I got a little carried away. I'll change this.

File: README.md (hunk 1 above)
Comment (degville) [reply README.md]: Good idea, thanks.

File: README.md (hunk 2 above)
Comment (degville) [reply README.md]: After a quick chat, we've decided to keep this as snappy for now.

**Comment by degville** [comment]
@anonymouse64 thanks for the review. I've pushed a couple of fixes.

File: README.md [hunk 4]
```diff
@@ -1,38 +1,68 @@
...
+[installation](https://snapcraft.io/docs/installing-snapd) and [Getting
+started](https://snapcraft.io/docs/getting-started) guides, head over to our
+[Snap documentation](https://snapcraft.io/docs). If you're looking for
+something to install, such as [Spotify](https://snapcraft.io/spotify) or
+[Visual Studio Code](https://snapcraft.io/code), take a look at the [Snap
+Store](https://snapcraft.io/store). And if you want to build your own snaps,
+start with our [Creating a snap](https://snapcraft.io/docs/creating-a-snap)
+documentation.
 
 [![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/snapd)
```
Comment (bboozzoo) [inline README.md]: While at it, I'm not sure we should have a link to the snapd page in snap store as snapd actually comes as a Linux distro package.

File: README.md [hunk 5]
```diff
@@ -1,38 +1,68 @@
...
+Snaps are app packages for desktop, cloud and IoT that update automatically,
+are easy to install, secure, cross-platform and dependency-free. They're being
+used on millions of Linux systems every day.
+
+Alongside its various service and management functions, snapd:
+- provides the _snap_ command that's used to install and remove snaps and
+  interact with the wider snap ecosystem
+- implements the confinement policies that isolate snaps from the base system
+  and from each other
+- marshals the interfaces that allow snaps to access specific system resources
```
Comment (zyga) [inline README.md]: I looked up _to marshal_ in a dictionary and while it does make sense I think it's somewhat farther away on the simple english spectrum. Perhaps this word could be replaced by _governs_?

File: README.md (hunk 4 above)
Comment (degville) [reply README.md]: Yeah, I agree with this.

File: README.md (hunk 5 above)
Comment (degville) [reply README.md]: Yes, I think you're right. Governs is a better/clearer word for this.

**Pull request merged**